modules/govuk_jenkins/templates/jobs/copy_data_from_staging_to_aws.yaml.erb

```diff
@@ -41,7 +41,7 @@
         numToKeep: 10
     builders:
         - shell: |
-            set +x
+            set -eu
```

Comment:
See this comment https://github.com/alphagov/govuk-puppet/pull/7409/files#r181757415